airline-web/app/controllers/Application.scala

```diff
@@ -509,6 +509,39 @@ class Application @Inject()(cc: ControllerComponents, val configuration: play.ap
     }))
   }
 
+  def getLinksByAirplaneModel(modelId : Int) = Action {
+    val linksByModel = LinkSource.loadFlightLinksByAirplaneModel(airportId).groupBy(_.from)
+
+    val sortedLinks: Map[List[Airport], List[Link]] = links.foldLeft(Map.empty[List[Airport], List[Link]]) { (map, link) =>
```

Comment:
@Bohaska the `links` should probably be linksByModel ?

Reply:
yes, this is also a bug